### ISSUE TITLE
RSS link points to nowhere

### ISSUE THREAD

**Issue by rethab** [kentcdodds/old-kentcdodds.com#384]
hi, the rss link on https://kentcdodds.com/ points to https://kentcdodds.com/blog/rss.xml which returns a 404.

**Comment by kentcdodds** [comment]
Ah, yeah, [I removed it](https://github.com/kentcdodds/kentcdodds.com/commit/f2f53488ade457881209a72b24f845745c75506c) but forgot to remove those references. Thanks!

**Comment by rethab** [comment]
Okay, fair enough. Although I'd like to add that this is very unfortunate, because it means I can no longer follow your blog using my RSS reader :(

**Comment by NicholasMurray** [comment]
Hi, Kent please reconsider, I don't know what hits you get on your rss feed but like @rethab rethab I find it hard to follow multiple blogs without a RSS reader feed.

**Comment by kentcdodds** [comment]
Yeah, I know is inconvenient. If you'd really like to have it back then you could figure out why gatsby-plugin-feed is so slow and speed it up. Unfortunately I don't have the bandwidth to work on it and I can't include it when it's so slow. Sorry.

**Comment by NicholasMurray** [comment]
Fair enough. NP.
